jobs/mozillians_magic.rb

```diff
@@ -42,19 +41,22 @@ def query_mozillians(params = {})
       http.request(request)
     end
 
-    def query_mozillians_group(group)
-      response = query_mozillians({ group: group })
-
-      if response.code.to_i == 200
-        res = JSON.parse(response.body)
-        count = res['count']
+    def iterate_mozillians_groups(groups)
+      remove_from_group('mozillians_nda')
```

Comment:
Probably it's best not to hardcode the value here. How about adding this to a var accessible from everywhere? This way if ever need to change the name, you can do so only in one place instead of changing multiple files.